implementations/elixir/ockam/ockam_healthcheck/README.md

```diff
@@ -1,28 +1,50 @@
 ## Ockam healthcheck
 
 This application runs periodic healthcheck requests to ockam nodes by establishing
-a secure channel and sending a ping message to the configured worker.
+a secure channel and:
+* sending a ping message to the configured worker for default targets
+* sending an Ockam API request with a specified path, method and optionally a body for API endpoint targets, where `200` response is considered a healthy result
 
 Healthcheck results are reported as prometheus metrics using `ockam_metrics` and
 as logs using Logger.
 
 ## Configuration
 
-There are two main configurations:
-
-- frequency
-  Configured as: `:ockam_healthcheck, :crontab` application environment or `HEALTHCHECK_CRONTAB` environment vatiable
-  Format: string in crontab format
-- healthcheck targets
-  Configured as: `:ockam_healthcheck, :targets` application environment or `HEALTHCHECK_TARGETS` environment vatiable
-  Format: application environment is a list of maps `[%{name: ..., host: ..., port: ..., api_worker: ..., healthcheck_worker: ...}, ...]`, environment variable is a JSON string with the same format
-
-For each target, the application will connect via tcp `host:port` connection, establish secure channel
-using `api_worker` listener and send ping to `healthcheck_worker`.
-
-Each cycle of crontab frequency the targets configuration will be read from the `:ockam_healthcheck, :targets`, which allows Elixir node to control targets.
-
-There is no way currently to control the frequency other than restarting the application.
+Healthcheck targets and frequency of calling them are configured as: `:ockam_healthcheck, :targets` application environment or `HEALTHCHECK_TARGETS` environment variable
+Format: application environment is a list of maps. For default (ping) targets the map is formatted as follows:
+```elixir
+%{
+  name: ...,
+  host: ...,
+  port: ...,
+  api_worker: ...,
+  healthcheck_worker: ...,
+  crontab: ...
+}
+```
+For API endpoint targets:
```

Comment:
Should we name `Ockam.Services.API` explicitly for clarity?

Reply:
Done in https://github.com/build-trust/ockam/pull/5078/commits/a23dafbc12b9fb12be1dcebe2a75921eb9decc59